### PR TITLE
Apply new influxdb schema

### PR DESCRIPTION
**Pull request by TheKK**
This is the schema we discuss with Eli.

- B2G on flame
benchmarks,bench-name=smoketest,name=worked,device=flame,platform=b2g,browser-version=45.0,os-version=2.6.0.0-prerelease,processor=armeabi-v7a value=1 1448933933530711808

- firefox on GNU/Linux
benchmarks,bench-name=smoketest,name=worked,device=desktop,platform=linux,browser-version=42.0,os-version=Ubuntu\ 14.04,processor=x86_64 value=1 1448934082006878976

Let me know if there's anything could be improved.

### CODE REVIEW

File: mozbench/resultRecorder.py [hunk 1]
```diff
@@ -68,14 +72,25 @@ def get_influxdb_results(self):
...
+                               ',device=' + device +
+                               ',platform=' + platform +
+                               ',browser-version=' + browser_version +
+                               ',os-version=' + osVersion +
+                               ',processor=' + processor)
+                        # Measurement names, tag keys, and tag values must escape any spaces using a backslash.
+                        tag = tag.replace(' ', '\ ') # TODO: comma and equal should be handled as well
+
+                        if isinstance(value, float):
+                            val = 'value=' + str(value)
```
Comment (dminor) [inline mozbench/resultRecorder.py]: 'value=%s' % value

File: mozbench/resultRecorder.py [hunk 2]
```diff
@@ -68,14 +72,25 @@ def get_influxdb_results(self):
...
+                        else:
+                            val = 'value=' + str(int(value))
```
Comment (dminor) [inline mozbench/resultRecorder.py]: 'value=%d' % value

File: mozbench/mozbench.py [hunk 3]
```diff
@@ -363,10 +365,12 @@ def cli(args):
         device = device_manager.get_prop('ro.product.device')
     elif use_android:
         platform = 'android'
-        device = mozdevice.ADBAndroid(args.device_serial)
+        device_manager = mozdevice.ADBAndroid(args.device_serial)
         os_version = device.get_prop('ro.build.version.release')
```
Comment (dminor) [inline mozbench/mozbench.py]: device_manager

Comment (dminor) [inline mozbench/mozbench.py]: device_manager.get_prop

File: mozbench/mozbench.py [hunk 4]
```diff
@@ -363,10 +365,12 @@ def cli(args):
...
         processor = device.get_prop('ro.product.cpu.abi')
```
Comment (dminor) [inline mozbench/mozbench.py]: device_manager.get_prop

File: mozbench/mozbench.py [hunk 5]
```diff
@@ -234,9 +234,10 @@ def postresults(logger, results):
     # we'll try four times before giving up
     for i in xrange(0, 4):
         try:
-            influxdb_url = url + '/db/' + dbtable + '/series?'
-            r = requests.post(influxdb_url + 'u=' + user + '&p=' + passwd,
-                              data=json.dumps(results))
+            headers = {'X-Requested-With': 'Python requests', 'Content-type': 'text/xml'}
+            influxdb_url = 'http://' + url + '/write?db=' + dbtable
```
Comment (dminor) [inline mozbench/mozbench.py]: I'd prefer to have 'http://' be part of the url so we also use 'https://'

**Comment by dminor** [comment]
Looks good!

File: mozbench/resultRecorder.py (hunk 1 above)
Comment (TheKK) [inline mozbench/resultRecorder.py]: This is clever solution, thanks!

**Pull request merged**